trulens_eval/trulens_eval/feedback/provider/base.py

```diff
@@ -133,11 +136,12 @@ def generate_score(
         self,
         system_prompt: str,
         user_prompt: Optional[str] = None,
-        normalize: float = 10.0,
+        min_score_val: int = 0,
```

Comment:
update the docstring to match new arguments

Reply:
thanks for catching this - done